sd_dynamic_prompts/dynamic_prompting.py

```diff
@@ -103,8 +105,8 @@ def __init__(self):
         if loaded_count % 2 == 0:
             return
 
-        callbacks.register_pnginfo_saver(self._pnginfo_saver)
-        callbacks.register_prompt_writer(self._prompt_writer)
+        # callbacks.register_pnginfo_saver(self._pnginfo_saver)
+        # callbacks.register_prompt_writer(self._prompt_writer)
```

Comment:
The prompt writer is unrelated to PNG info, it just writes prompts into CSV files.
```suggestion
        callbacks.register_prompt_writer(self._prompt_writer)
```

Reply:
my bad